sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample03_DetectionConfigurationCrudOperations.cs

```diff
@@ -13,6 +13,7 @@ namespace Azure.AI.MetricsAdvisor.Samples
     public partial class MetricsAdvisorSamples : MetricsAdvisorTestEnvironment
     {
         [Test]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21663")]
         public async Task CreateAndDeleteDetectionConfigurationAsync()
```

Comment:
I didn't want to make samples depend on mock entities in the service. I want to keep them easy for users to run. We may need to create a separate data feed for .NET exclusive use, but I'll do this in another PR as it may trigger too many changes in tests/recordings. Re-enabling the sample is tracked with https://github.com/Azure/azure-sdk-for-net/issues/21663.